src/index.ts

```diff
@@ -327,6 +342,83 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     return this.sendRawBundle(signedTransactions, targetBlockNumber, opts)
   }
 
+  public async sendPrivateTransaction(
+    transaction: FlashbotsBundleTransaction,
+    opts?: {
+      maxBlockNumber?: number,
+      minTimestamp?: number,
+    },
+  ): Promise<FlashbotsPrivateTransaction> {
+    return this.sendPrivateRawTransaction(
+      await transaction.signer.signTransaction(transaction.transaction),
+      opts,
+    )
+  }
+
+  public async sendPrivateRawTransaction(
```

Comment:
My thinking here was to adopt the pattern of `sendBundle` and `sendRawBundle` and keep things consistent.

Technically the `eth_sendPrivateTransaction` RPC method does take raw transactions. `sendPrivateTransaction` is just a wrapper for `sendPrivateRawTransaction`.

I think having both gives the user more freedom. Some users might prefer the `FlashbotsBundleTransaction` interface:
```js
sendPrivateTransaction({
  transaction: {...},
  signer: myWallet,
})
```

Some might prefer to sign themselves and use the raw tx:
```js
const signedTx = myWallet.signTransaction({...})
sendPrivateRawTransaction(signedTx)
```

Thoughts?

Reply:
To me it's confusing because `sendPrivateRawTransaction` is a specific API that's introduced in mev-geth v0.6, which lets miners accept private transactions. But this change here introduces a `sendPrivateRawTransaction` with different mechanics (namely sending the tx with `sendPrivateTransaction` to the relay).

Reply:
Ok, makes sense. I removed `sendPrivateRawTransaction` and modified `sendPrivateTransaction` to accept both flashbots tx types. What do you think?